owncloudApp/src/main/java/com/owncloud/android/presentation/ui/security/PassCodeActivity.kt

```diff
@@ -282,10 +290,38 @@ class PassCodeActivity : AppCompatActivity() {
         }
     }
 
+    private fun lockScreen() {
+        val timeToUnlock = passCodeViewModel.getTimeToUnlockLeft()
+        if (timeToUnlock > 0) {
+            binding.lockTime.isVisible = true
+            for (editText: EditText? in passCodeEditTexts) {
+                editText?.isEnabled = false
+            }
+            object : CountDownTimer(timeToUnlock, 1000) {
+                override fun onTick(millisUntilFinished: Long) {
+                    val minutes = ((millisUntilFinished + 1000) / 1000) / 60
+                    val seconds = ((millisUntilFinished + 1000) / 1000) % 60
```

Comment:
Try using Kotlin's binary operators instead of expressions, you can check it here: https://kotlinlang.org/docs/operator-overloading.html#binary-operations

Reply:
Done!